contrib/sip_proxy/filters/network/source/router/router_impl.cc

```diff
@@ -179,7 +182,7 @@ FilterStatus Router::handleAffinity() {
   if (metadata->pCookieIpMap().has_value()) {
     auto [key, val] = metadata->pCookieIpMap().value();
     ENVOY_LOG(trace, "update p-cookie-ip-map {}={}", key, val);
-    callbacks_->traHandler()->updateTrafficRoutingAssistant("lskpmc", key, val);
+    callbacks_->traHandler()->updateTrafficRoutingAssistant("lskpmc", key, val, absl::nullopt);
```

Comment:
Here and in line 699, when called updateTrafficRoutingAssistant() function, as it has the optional null parameter for the SIP context, I modified the call but passing an ```absl::nullopt``` value. If you prefer to change it, let me know